models/public/gmcnn-tf/gmcnn-tf.md

```diff
@@ -0,0 +1,146 @@
+# gmcnn-tf
+
+## Use Case and High-Level Description
+
+The `gmcnn-tf` is the TensorFlow implementation of GMCNN Image Inpainting model,
+aimed to estimate suitable pixel information tofill holes in images. The `gmcnn-tf`
+is trained on Places2 dataset with free-form masks. Originally redistributed as checkpoint files,
+was converted to frozen graph.For details see repository <https://github.com/shepnerd/inpainting_gmcnn>.
+
+### Steps to Reproduce Conversion to Frozen Graph
+
+1. Clone the original repository
+```sh
+git clone https://github.com/shepnerd/inpainting_gmcnn.git
+cd inpainting_gmcnn/tensorflow
+```
+2. Checkout the commit that the conversion was tested on:
+```sh
+git checkout ba7f710
+```
+3. Install the [original dependencies](https://github.com/shepnerd/inpainting_gmcnn#prerequisites).
+(TensoFlow\* version used - 1.14.0, CPU).
+4. Download the [pretrained weights](https://drive.google.com/file/d/1aakVS0CPML_Qg-PuXGE1Xaql96hNEKOU/view?usp=sharing)
+5. Run sample [conversion script](./freeze_model.py):
```

Comment:
Something seems to be missing here. If you run the script as written, it's not going to work, since `sys.path` does not, by default, include the current directory.